lua/treesitter-context.lua

```diff
@@ -132,7 +127,13 @@ function M.enable()
 
   attached[cbuf] = true
 
-  autocmd({ 'WinScrolled', 'BufEnter', 'WinEnter', 'VimResized' }, update)
+  autocmd({ 'BufEnter', 'WinEnter', 'VimResized' }, update)
+
+  autocmd({ 'WinScrolled' }, function()
+    for winid, bufnr_map in pairs(require('treesitter-context.render').get_bufnr_maps()) do
+      update(bufnr_map.bufnr, winid)
+    end
+  end)
```

Comment:
Are we updating all the windows on scroll? Shouldn't we be only updating the scrolled one?

Reply:
Commands such as split will trigger winscroll, and every window width should be adjusted to new width, how should we differentiate scroll and resizing? Should I use winresized instead?